python/main.py

```diff
@@ -98,8 +111,66 @@ async def get_image(image_name):
 class Item(BaseModel):
     name: str
     category: str
+    image_name: str
 
 
-def insert_item(item: Item):
+def insert_item(item: Item, image_name: str):
```

Comment:
We are already adding `image_filename` to `Item` above, this seems to be a duplicate?